Arch Linux Unsupported?

I am actually trying to install mockgoose, but it looks like the issues come all the way back to this package. When I attempt to run `npm install mongodb-prebuilt`, I get the following output:

```
> mongodb-prebuilt@4.2.16 postinstall /usr/lib/node_modules/mongodb-prebuilt
> node install.js

/usr/lib/node_modules/mongodb-prebuilt/node_modules/mongodb-download/index.js:130
                throw new Error("unsupported linux distribution " + JSON.stringify(os));
                ^

Error: unsupported linux distribution {"os":"linux","dist":"Arch Linux"}
    at /usr/lib/node_modules/mongodb-prebuilt/node_modules/mongodb-download/index.js:130:11
    at /usr/lib/node_modules/mongodb-prebuilt/node_modules/getos/index.js:65:18
    at customLogic (/usr/lib/node_modules/mongodb-prebuilt/node_modules/getos/index.js:112:13)
    at /usr/lib/node_modules/mongodb-prebuilt/node_modules/getos/index.js:62:16
    at tryToString (fs.js:414:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:401:12)
npm ERR! Linux 4.2.5-1-ARCH
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "-g" "mongodb-prebuilt"
npm ERR! node v5.3.0
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE

npm ERR! mongodb-prebuilt@4.2.16 postinstall: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the mongodb-prebuilt@4.2.16 postinstall script 'node install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the mongodb-prebuilt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs mongodb-prebuilt
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls mongodb-prebuilt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/dtaylor/Development/hapi-api-plugin-v2/npm-debug.log
```

[npm-debug.zip](https://github.com/winfinit/mongodb-download/files/66320/npm-debug.zip)

--- a/index.js
+++ b/index.js
@@ -72,7 +72,7 @@ module.exports = function (opts, cb) {
 
 	var name = "mongodb-" + mongo_platform + "-" + mongo_arch;
 
-	if ( mongo_platform === "linux" ) {
+	if ( mongo_platform === "linux" && mongo_arch !== "i686" ) {
 		// append distro
 		getos(function(e, os) {
 			if(e) {

